homeassistant/components/qingping/sensor.py

```diff
@@ -69,18 +71,18 @@
         QingpingSensorDeviceClass.PM10,
         Units.CONCENTRATION_PARTS_PER_MILLION,
```

Comment:
Should this also change?

Reply:
Yes that would help.  I did get it right in the library at least

616bd742043f3c2877d6e1f0e50cf92dff4ff910

Reply:
Thanks